### PR TITLE
Feature/edit button

### PR DESCRIPTION
**Pull request by lbayliss-scottlogic**

### CODE REVIEW

File: src/components/elements/EditableDiv.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import "bootstrap-icons/font/bootstrap-icons.css";
+import classes from "./EditableDiv.module.css";
+
+function EditableDiv(props) {
```
Comment (DKDiep) [inline src/components/elements/EditableDiv.js]: I wouldn't tie the name of a component to implementation detail (div), maybe use something more generic?

File: src/components/elements/EditableDiv.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      `https://friendface-react-default-rtdb.europe-west1.firebasedatabase.app/posts/${props.data.id}.json`,
+      {
+        method: "PATCH",
+        body: JSON.stringify(postData),
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }
+    );
+    // TODO: Feedback to the user that the post change was successful .then() ?
```
Comment (DKDiep) [inline src/components/elements/EditableDiv.js]: Why can't this be done as a part of this task/PR? Unless it is a large ordeal or that we're pressed by deadlines, I think we should finish the feature properly

File: src/components/posts/PostItem.js [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
+import EditableDiv from '../elements/EditableDiv';
 
 function PostItem(props) {
   return (
     <Post>
       <div className={classes.containerHeader}>
         <span className={classes.author}>{props.author}</span>
         <span>{props.likes}</span>
-        <LikeButton id={props.id}/>
+        <LikeButton id={props.id} />
```
Comment (DKDiep) [inline src/components/posts/PostItem.js]: I see some self closing elements have a space and some don't, it's better to be consistent: which one do you want to stick with?

File: src/components/elements/EditableDiv.js (hunk 1 above)
Comment (lbayliss-scottlogic) [reply src/components/elements/EditableDiv.js]: Changed to 'Box' instead

File: src/components/posts/PostItem.js (hunk 3 above)
Comment (lbayliss-scottlogic) [reply src/components/posts/PostItem.js]: I believe anytime I don't use any innerHTML I've used self closing elements - will change anytime an exception to this.

**Review by DKDiep — APPROVED** [review]

File: src/pages/CreatePost.js [hunk 4]
```diff
@@ -6,8 +6,11 @@ function CreatePost() {
   const navigate = useNavigate();
 
   function newPostHandler(postData) {
+
+    console.log(JSON.stringify(postData));
```
Comment (DKDiep) [inline src/pages/CreatePost.js:10]: Was this left from debugging? If so we shouldn't include it in production code.

**Pull request merged**